test/cpp/test_xla_sharding.cpp

```diff
@@ -1,59 +1,67 @@
 #include <ATen/ATen.h>
 #include <gtest/gtest.h>
+#include <stdlib.h>
 
 #include <iostream>
 
 #include "cpp_test_util.h"
-#include "tensorflow/compiler/xla/client/xla_builder.h"
-#include "tensorflow/compiler/xla/shape_util.h"
-#include "tensorflow/compiler/xla/xla_client/computation_client.h"
-#include "tensorflow/compiler/xla/xla_client/debug_macros.h"
-#include "tensorflow/compiler/xla/xla_client/multi_wait.h"
-#include "tensorflow/compiler/xla/xla_client/thread_pool.h"
+#include "tensorflow/compiler/xla/xla_client/env_vars.h"
+#include "tensorflow/compiler/xla/xla_client/sys_util.h"
+#include "tensorflow/compiler/xla/xla_data.pb.h"
 #include "torch_xla/csrc/aten_xla_bridge.h"
 #include "torch_xla/csrc/helpers.h"
+#include "torch_xla/csrc/tensor.h"
 #include "torch_xla/csrc/tensor_util.h"
-#include "torch_xla/csrc/torch_util.h"
+#include "torch_xla/csrc/xla_sharding_util.h"
 #include "torch_xla_test.h"
 
 namespace torch_xla {
 namespace cpp_test {
-
-// A placeholder for future SPMD sharding tests.
+namespace {
+bool TwoEqualXlaData(torch::lazy::BackendDataPtr a,
```

Comment:
ditto

Reply:
Sorry for missing this. Just addressed this.

Reply:
Done.